src/ElmHtml/InternalTypes.elm

```diff
@@ -6,6 +6,7 @@ module ElmHtml.InternalTypes
         , CustomNodeRecord
         , MarkdownNodeRecord
         , Facts
+        , Tagger
```

Comment:
there's no need for this new type, this will bump the version without needing it